File: vulcanus/src/main/java/com/merkury/vulcanus/model/dtos/account/add/spot/AddSpotDto.java
package com.merkury.vulcanus.model.dtos.account.add.spot;

import com.merkury.vulcanus.model.embeddable.BorderPoint;
import lombok.Builder;

import java.util.List;

@Builder
public record AddSpotDto(Long id,
                         String name,
                         String description,
                         String country,
                         String region,
                         String city,
                         String street,
                         List<BorderPoint> borderPoints,
                         String firstPhotoUrl) {
}

File: venus/src/model/interface/account/add-spot/addSpotDto.ts
import SpotCoordinatesDto from "../../spot/coordinates/spotCoordinatesDto";

export interface AddSpotDto {
    id: number;
    name: string;
    description: string;
    country: string;
    region: string;
    city: string;
    street: string;
    borderPoints: SpotCoordinatesDto[];
    firstPhotoUrl: string;
}
